src/full/Agda/Utils/Empty.hs

```diff
@@ -1,12 +1,11 @@
-{-# OPTIONS_GHC -Wunused-imports #-}
```

Comment:
Maybe there is a misunderstanding here.
This pragma turns _on_ the warning about unused imports, thus, keeping the import lists exact.  I think it would help your goal to improve code clarity if that warning was left in place.
Note that the `unused-imports` warning is by default off, via a setting in `Agda.cabal`.  I recently turned it on for all modules by prepending the pragma to all files, and only removed it from these where it would be too cumbersome to address the warnings.  (Sometimes newer versions of `Prelude` export more, triggering warnings on newer GHC versions.)

Reply:
Thank you for the feedback. I have reinstated the `-Wunused-imports` pragma as advised. I've also made some additional edits to files in the utilities directory.

Reply:
Oh, lots of errors. I will have to check slower.